### DISCUSSION TITLE
pattern Secret Scanning progress

### DISCUSSION THREAD

**Discussion by rafik-sg** [community/community#32310] [Enterprise]
Hello,

We are using Secret Scanning with Github Enterprise.
When we create a new pattern or update  an old one, we have noticed that the modification takes time to become effective either to scan repositories or to delete the no longer valid alerts. And there is no way to check what has been sacanned and what has not.
Is there a way to get the secret scanning progress ?

**Comment by 15MariamS** [comment]
Hi @rafik-sg, 

We're working on making this experience better. I have a few questions to help me better understand your pain point:
- Are you working mainly at the organization level?
- If you were looking at scan status at that level, what would you want to see (for example, which repos were scanning? when the last scan took place?)
- Would you expect an email notification informing you that the entire scan completed?

**Comment by srkambham** [comment]
Hi @15MariamS,
I am currently facing the above issue as well.
I have just added one more custom pattern and published at organization level to further cut down false positives.
Earlier when I had one, I had open secrets as around 2000 , but after publishing another one now I see them as about 5000.
I would like to understand, whether the scan is still going on or not ? its been more than an hour.
Yes, we would like to see the status/progress of the scan

Please suggest.

**Reply by 15MariamS** [reply]
Noted, and this is on our roadmap. If you can also answer the [questions above](https://github.com/orgs/community/discussions/32310#discussioncomment-3965762), I'd love to keep your feedback in mind when we build a solution!

**Comment by rafik-sg** [comment]
Hi @15MariamS,

Sorry for the late response.

- I am mainly  working on Enterprise level
- The most important information is the completion of organization/Enteprise scanning for a given pattern, something like status tag over the pattern. It would be nice to have a status by organization for enterprise level or per repository for organization level.
- An email notification would be nice too.
